libswc/swc.h

```diff
@@ -27,6 +27,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 #include <wayland-server.h>
+#include <libinput.h>
```

Comment:
Just use a forward declaration of `struct libinput_device` instead of this include.